client/lib/purchases/index.js

```diff
@@ -212,6 +212,38 @@ function handleRenewMultiplePurchasesClick( purchases, siteSlug, options = {} )
 	page( renewalUrl );
 }
 
+/**
+ * Adds a purchase renewal to the cart and redirects to checkout.
```

Comment:
I'm a little confused by this function; it references "renewals" quite a bit but the URL it constructs is for adding a new product, not for adding a renewal. You can see the structure for a renewal product URL a few lines above (`/checkout/${slug}/renew/${purchaseId}/${siteSlug}`).

I know this PR has something to do with downgrades, so maybe in that case you actually want a new product and not a renewal (since if you change plans, it's a different plan). If that's the case, though, this function should not use "renewal" because it might confuse other folks who use it in the future.

It's very likely that I'm misunderstanding something, though, so please help me to understand!